src/components/routes/Trips/ActiveTripCard.tsx

```diff
@@ -40,7 +41,7 @@ const ActiveTripCard = ({ onCancelClick, trip }: Props) => {
         <div className="address">
         <BeeLink href={getGoogleMapURI(listing)} target="_blank">
           <ListItem noHover suffixColor="secondary" textColor="secondary" textTransform="uppercase">
-            <span>{formatAddress(streetAddress, listing.city, listing.state, listing.country.toUpperCase())}</span>
+            <span>{formatAddress(streetAddress || `Lat, Lng: ${lat}, ${lng}`, listing.city, listing.state, listing.country.toUpperCase())}</span>
```

Comment:
i think theres a `formatGeolocationAddress` you can use

Reply:
🙏